rllib/algorithms/algorithm.py

```diff
@@ -1083,7 +1083,15 @@ def duration_fn(num_units_done):
                         for ma_batch in batches:
                             ma_batch = ma_batch.as_multi_agent()
                             for batch in ma_batch.policy_batches.values():
-                                assert batch.is_terminated_or_truncated()
+                                if not batch.is_terminated_or_truncated():
+                                    raise ValueError(
+                                        "RLlib attempts to evaluate your policy per "
+                                        "episode, which is determined by the "
+                                        "evaluation config's "
+                                        "`evaluation_duration_unit`. However, "
+                                        "the evaluation batch is neither terminated "
+                                        "nor truncated."
```

Comment:
```suggestion
                                        "In your evaluation config you have specified evaluation_duration_unit="episodes". However, the evaluation batch is neither terminated nor truncated."
```
This covers the reason of failure, but what is the remedy for this? What should we tell the user next so that they will be unblocked?